provisioning/roles/sp_mini_7_setup_init/tasks/main.yml

```diff
@@ -9,29 +9,35 @@
     dest: "/etc/init.d"
     mode: 0755
   with_items:
-    - kibana4_init
     - snowplow_mini_control_plane_api
-    - snowplow_stream_collector
-    - snowplow_stream_enrich
-    - snowplow_elasticsearch_loader_good
-    - snowplow_elasticsearch_loader_bad
-    - iglu_server_0.2.0
     - caddy_init
-    - nsqd_init
-    - nsqlookupd_init
 
 - name: Configure for inits for calling at boot time
   become: yes
   shell: "update-rc.d {{item}} defaults"
   with_items:
-    - kibana4_init
     - snowplow_mini_control_plane_api
-    - snowplow_stream_collector
-    - snowplow_stream_enrich
-    - snowplow_elasticsearch_loader_good
-    - snowplow_elasticsearch_loader_bad
-    - iglu_server_0.2.0
-    - elasticsearch
     - caddy_init
-    - nsqd_init
-    - nsqlookupd_init
+
+- name: Start UI and Control Plane
+  become: yes
+  shell: service snowplow_mini_control_plane_api start && service caddy_init start
+
+- name: Increase mmap count to recommended 262144 for Elasticsearch
+  become: yes
+  shell: echo "vm.max_map_count=262144" >> /etc/sysctl.conf && service procps start
+
+- cron:
+    name: "Add cronjob to restart containers at system reboot"
+    special_time: reboot
+    job: /usr/local/bin/docker-compose -f /home/ubuntu/snowplow/docker-compose.yml restart && sleep 30
+
+- cron:
+    name: "Add cronjob to crop ES good index's docs older than a week"
+    special_time: weekly
+    job: /usr/bin/curl -s -X POST http://localhost:9200/good/_delete_by_query -d '{ "query" :{ "range" :{ "collector_tstamp" :{ "lt" :"now-1w/d" } } } }' > /dev/null 2>&1
+
+- cron:
```

Comment:
can we have something more specific as far as names are concerned instead of 3 times `cron`?